test/lsp/lsp_hover.ml

```diff
@@ -143,3 +195,850 @@ let%expect_test "hover-replaced" =
+    (line 12, character 14):
+    __rootdir__/prog.cob:13.14-13.20:
+      10               88 COND-2 VALUE "a".
+      11           PROCEDURE DIVISION.
+      12             SET COND-1 TO TRUE
+      13 >           SET COND-2 TO TRUE
+    ----                 ^^^^^^
+      14             STOP RUN.
+      15
+    ```cobol
+    88 COND-2 VALUE "a".
+    ```
+    ```cobol
+    VAL-2 IN STRUCT
+    ```
+    USAGE DISPLAY
+    ALPHANUMERIC(1) |}];;
+
+let%expect_test "hover-typedef-renames" =
+  let { projdir; end_with_postproc }, server = make_lsp_project () in
+  print_hovered server ~projdir @@ extract_position_markers {cobol|
+        IDENTIFICATION DIVISION.
+        PROGRAM-ID. prog.
+        DATA DIVISION.
+        WORKING-STORAGE SECTION.
+        01 X.
+          05 Y PIC 9.
+          05 YY PIC XX.
+          66 Z_|_ R_|_ENAMES _|_Y.
+          66 Y-THRU-YY_|_ RENAMES Y THRU YY.
+        PROCEDURE DIVISION.
+            DISPLAY _|_Z.
+            STOP RUN.
+    |cobol};
+  end_with_postproc [%expect.output];
+  [%expect {|
+    {"params":{"diagnostics":[],"uri":"file://__rootdir__/prog.cob"},"method":"textDocument/publishDiagnostics","jsonrpc":"2.0"}
+    (line 8, character 14):
+    __rootdir__/prog.cob:9.13-9.14:
+       6           01 X.
+       7             05 Y PIC 9.
+       8             05 YY PIC XX.
+       9 >           66 Z RENAMES Y.
+    ----                ^
+      10             66 Y-THRU-YY RENAMES Y THRU YY.
+      11           PROCEDURE DIVISION.
+    ```cobol
+    Z IN X
+    RENAMES Y IN X
+    ```
+    USAGE DISPLAY
+    NUMERIC(digits = 1, scale = 0, with_sign = false)
+    (line 8, character 16):
+    __rootdir__/prog.cob:9.10-9.25:
+       6           01 X.
+       7             05 Y PIC 9.
+       8             05 YY PIC XX.
+       9 >           66 Z RENAMES Y.
+    ----             ^^^^^^^^^^^^^^^
+      10             66 Y-THRU-YY RENAMES Y THRU YY.
+      11           PROCEDURE DIVISION.
+    ```cobol
+    Z IN X
+    RENAMES Y IN X
+    ```
+    USAGE DISPLAY
+    NUMERIC(digits = 1, scale = 0, with_sign = false)
+    (line 8, character 23):
+    __rootdir__/prog.cob:9.23-9.24:
+       6           01 X.
+       7             05 Y PIC 9.
+       8             05 YY PIC XX.
+       9 >           66 Z RENAMES Y.
+    ----                          ^
+      10             66 Y-THRU-YY RENAMES Y THRU YY.
+      11           PROCEDURE DIVISION.
+    ```cobol
+    Y IN X
+    ```
+    USAGE DISPLAY
+    NUMERIC(digits = 1, scale = 0, with_sign = false)
+    (line 9, character 22):
+    __rootdir__/prog.cob:10.13-10.22:
+       7             05 Y PIC 9.
+       8             05 YY PIC XX.
+       9             66 Z RENAMES Y.
+      10 >           66 Y-THRU-YY RENAMES Y THRU YY.
+    ----                ^^^^^^^^^
+      11           PROCEDURE DIVISION.
+      12               DISPLAY Z.
+    ```cobol
+    Y-THRU-YY IN X
+    RENAMES Y IN X
+    THRU YY IN X
+    ```
+    USAGE DISPLAY
+    ALPHANUMERIC(24)
```

Comment:
@NeoKaios Could you amend the commit that promoted tests to fix this line (via `git rebase -i master`)?